### PR TITLE
Add a new menu 'Meetings' for meeting/workshop summaries

### PR DESCRIPTION
**Pull request by jouvin**
This is an attempt to migrate workshop summaries from TracWiki.
In the initial commit something doesn't work properly: the menu is created but the posts belonging to the new category `meeting` are not displayed. Coud a Jekyll expert tell me what is missing in my declaration of the new category?

### CODE REVIEW

**Comment by jrha** [comment]
There is no [Jekyll frontmatter](http://jekyllrb.com/docs/frontmatter/) in your new post documents, without a category specified in the frontmatter, Jekyll will be unable to figure out where to put them.

**Comment by jrha** [comment]
Which is odd, because I see that your script makes allowances for the frontmatter.

**Comment by jouvin** [comment]
Thanks, I missed that due to probably a mistake of mine the frontmatter disappeared... I'll fix it.

**Comment by jouvin** [comment]
Should be ok now. About the title in the frontmatter, do you prefer the current `Quattor Workshop Summary` for all document or do you prefer `nth Quattor Workshop Summary`?

**Comment by jouvin** [comment]
I check the test errors. This is due to links in the documents that are no longer reachable. I'll try to clean them up.

**Comment by jrha** [comment]
The titles are what will appear in the list, so it would be good for them to be meaningful! For example our existing meeting notes state the number, dates and location of the meeting.

> Summary of 21th Quattor workshop (2016-03-22 to 2016-03-24, Bruxelles)

That might be a lot of work, so realistically whatever you have time to add would be great.

**Comment by jouvin** [comment]
I fixed/removed all broken links... The remaining error is caused by metacpan urls in `metaconfig/index.html`. I checked the URL but I have the feeling that metacpan is broken right now: no module can be found despite the full list of modules available in metacpan lists it... To be checked later maybe... @stdweird what do you think?

**Comment by jouvin** [comment]
This PR should be complete now. I fixed the page titles to reflect the workshop number.

**Comment by stdweird** [comment]
@jouvin the metacpan urls from the failing travis test all seem to work for me. if there still is an issue, maybe make a new issue here (since currently all tests seem to pass)
big thanks for this effort btw

**Comment by jouvin** [comment]
No, as expected, it was a transient problem with metacpan. I have been able to check that it was not answerin at the time the test failed but the problem was fixed in the next 15 minutes. This PR can be merged now, except if somebody has a remark on the contents.

File: _posts/2007-03-12-workshop.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: article
-title: Quattor Workshop Summary
+title: 3d Quattor Workshop Summary (March 2007)
```
Comment (jrha) [inline _posts/2007-03-12-workshop.md]: 3**r**d (hopefully they're all in 3D)

**Comment by jrha** [comment]
@jouvin please don't move the existing notes, it will break any external links pointing at them.
Take a look at #179 for a couple of suggestions on how to do it cleanly.

**Comment by jouvin** [comment]
Old urls restored using frontmatter `redirect_from`. Should be ready for merging.

Fixes #179

**Comment by jrha** [comment]
LGTM many thanks for doing this.

**Pull request merged**